domains/fbi.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "2Much4U's Website",
+  "repo": "https://github.com/2m4u",
+  "owner": {
+    "username": "2M4U",
+    "email": "2m4u@fbi.is-a.dev"
```

Comment:
maybe not use an is-a-dev ẹmail here, the email is used to call you when there is a problem with the domain, if your domain is broken how should we contact you?